discovery-provider/src/queries/get_health.py

```diff
@@ -197,7 +199,17 @@ def get_health(args, use_redis_cache=True):
     user_balances_age_sec = get_elapsed_time_redis(
         redis, user_balances_refresh_last_completion_redis_key
     )
-
+    num_users_in_balance_refresh_queue = len(
+        redis.smembers(REDIS_USER_BALANCE_REFRESH_KEY)
+    )
+    last_scanned_block_for_balance_refresh = redis_get_or_restore(
+        redis, eth_indexing_last_scanned_block_key
+    )
+    last_scanned_block_for_balance_refresh = (
```

Comment:
I think this is useful, but what would be really nice would be to include an elapsed seconds for when this last successfully ran without error (can use the `get_elapsed_time_redis` pattern), because with that we can easily set up monitoring, whereas last_scanned_block is difficult